index.dd

```diff
@@ -98,8 +96,14 @@ void main()
 ----
 )
 
+) $(COMMENT your-code-here)
+))) $(COMMENT intro, div, div)
+
+$(COMMENT
+Picking the frontpage example after the page has loaded leads to reflows
+and re-rendering. Hence, we pick the example as soon as possible
 )
-)))
+<script src="$(ROOT_DIR)js/example_roulette.js"></script>
```

Comment:
> Didn't you believe me, when I showed that using a separate file doesn't matter performance-wise?

1. It doesn't matter today
2. It doesn't matter in that browser
3. Not an excuse to make a sub-ideal situation worse. What that shows is that more things need to be loaded lazily.